aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/test_instrumentation_patch.py

```diff
@@ -93,6 +96,10 @@ def _test_unpatched_botocore_instrumentation(self):
         self.assertFalse("aws.sqs.queue_url" in attributes)
         self.assertFalse("aws.sqs.queue_name" in attributes)
 
+    def _test_unpatched_gevent_ssl_instrumentation(self):
+        # Ssl
+        self.assertFalse("gevent.ssl" in sys.modules, "Upstream has added the gevent ssl patch")
```

Comment:
How does checking `gevent.ssl` in sys.modules help verify that the `ssl` module was in fact actually patched by the `gevent.monkey` module? Is there a better way to check? Maybe use the [`is_module_patched()`](https://www.gevent.org/api/gevent.monkey.html#gevent.monkey.is_module_patched) method from gevent?

Also, the messaging is incorrect. This patching has nothing to do with the "Upstream" (meaning OTel). It is us that is actually doing the gevent patch.